src/llfs/data_packer.test.cpp

```diff
@@ -189,4 +189,52 @@ TEST(DataPackerTest, ParallelDataCopyFalse)
   run_parallel_copy_test(false, 0);
 }
 
+void run_pack_record_test(const usize count)
```

Comment:
Please add doc comment describing this test and the meaning of the `count` parameter.